OnBoardingCalculator/app/src/main/java/com/globant/onboardingcalculator/mvp/model/CalculatorModel.java

```diff
@@ -56,4 +58,33 @@ public String getSecondOperand() {
     public char getOperator() {
         return operator;
     }
+
+    public void operate() {
+        Double result;
+        switch (operator) {
+            case Constants.OPERATOR_PLUS:
```

Comment:
pls try to import Constants.OPERATOR_PLUS. So in the code you only need to use OPERATOR_PLUS

Reply:
Done!